sprocs/instance_questions_update_score.sql

```diff
@@ -74,6 +95,13 @@ BEGIN
         RAISE EXCEPTION 'found submission with id=%, but question does not match %', arg_submission_id, arg_qid;
     END IF;
 
+    IF arg_modified_at IS NOT NULL AND current_modified_at != arg_modified_at THEN
+        modified_since_query = TRUE;
```

Comment:
@nwalters512 do you think a grading log should be added in this case, even if unsuccessful?

Reply:
I don't think so - I think that'd be a bit confusing/misleading?

Reply:
what about a grading job?